docs/ref/macro.rst

```diff
@@ -174,87 +279,149 @@ plural
 
 .. code-block:: jsx
 
-   function plural(value: string | number, options: Object)
+   plural(value: string | number, options: Object)
 
 ``plural`` macro is used for pluralization, e.g: messages which has different form
-based on counter. It accepts an object with required key ``value`` which determines
-the plural form. The only required plural form is a catch-all ``other``. Other forms
-depends on source language you're using (e.g: English has ``one`` and ``other`` plural
-forms).
+based on counter. The first argument ``value`` determines the plural form.
+The second argument is an object with available plural forms. Plural form
+used in the source code depends on your source locale (e.g. English has only
+``one`` and ``other``).
 
 .. code-block:: jsx
 
-   import { setupI18n } from "@lingui/core"
    import { plural } from "@lingui/macro"
-
-   const i18n = setupI18n()
-
-   const msg = i18n._(plural(count, {
+   const message = plural(count, {
       one: "# Book",
       other: "# Books"
-   }))
+   })
+
+   // ↓ ↓ ↓ ↓ ↓ ↓
+
+   import i18n from "@lingui/core"
+   const message = i18n._(/*i18n*/{
+     id: '{count, plural, one {# Book} other {# Books}}',
+     values: { count }
+   })
+
+If you need to add variables to plural form, you can use template string literals.
+This time :jsmacro:`t` macro isn't required as template strings
+are transformed automatically:
 
-   // t macro isn't required for nested messages,
-   // template strings are transformed automatically.
-   const vars = i18n._(plural(count, {
+.. code-block:: jsx
+
+   import { plural } from "@lingui/macro"
+   const message = i18n._(plural(count, {
```

Comment:
is this i18n._. required?, on the codemod i'm replacing all the i18n._(plural()), for just plural(value, {})

Reply:
No, it's not 👍